iib/web/static/api_v1.yaml

```diff
@@ -626,16 +663,13 @@ components:
                 This determined what level of protection the addition or removal had.
               type: string
               example: 'prod'
-            organization:
-              type: string
-              example: openshift-operators
             removed_operators:
               type: array
               items:
                 type: string
               example:
                 - prometheus-operator
-    RequestAdd:
+    AddRequest: # TODO maybe optimalize Add and Rm
```

Comment:
Can I ask optimize in what sense?

Reply:
Add and RM request use similar strucuture. But on the second thought, I think we don't need to do anything with that.